packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -330,6 +330,7 @@ const ContractTypeWidget = observer(
                             <ContractType.Info
                                 handleSelect={handleSelect}
                                 item={item || { value }}
+                                selected_value={value}
```

Comment:
`<ContractType.Info />` component  doesn't determine changes inside of `item || { value }` because of `observer`. So, that is why passing an additional primitive value.